api/v3/openapi.yaml

```diff
@@ -322,6 +322,50 @@ paths:
         '200':
           $ref: '#/components/responses/ListConsumersResponse'
 
+  /clusters/{cluster_id}/consumer-groups/{consumer_group_id}/lag:
+    parameters:
+      - $ref: '#/components/parameters/ClusterId'
+      - $ref: '#/components/parameters/ConsumerGroupId'
+
+    get:
+      summary: 'Get Consumer Group Lag.'
+      description: 'Returns the max and total lag of the consumers belonging to the specified consumer group.'
+      tags:
+        - Consumer Group
+      responses:
+        '200':
+          $ref: '#/components/responses/GetConsumerGroupLagResponse'
+
+  /clusters/{cluster_id}/consumer-groups/{consumer_group_id}/topics/{topic_name}/partitions/{partition_id}/lag:
```

Comment:
@rigelbm I think we discussed having the path be `/clusters/{cluster_id}/topics/{topic_name}/partitions/{partition_id}/consumer-groups/{consumer_group_id}lag` instead, but after considering how to define the path for List Consumer Lags, this ordering seemed to make more sense to me.

Reply:
* `/clusters/{cluster_id}/topics/{topic_name}/partitions/{partition_id}/lags`
List of all the lags for a given topic-partition

{
  "data": [
     {
        "metadata": { link },
        "cluster_id"
        "topic_name"
        "partition_id"
         ...
        "lag"...
     },
  ]
}


* `/clusters/{cluster_id}/topics/{topic_name}/partitions/{partition_id}/lags/{consumer_group_id}`
Specific lag for given topic-partition/consumer-group

{
  "metadata": { link },
  "cluster_id"
  "topic_name"
  ...
  "lag"...
}


* `/clusters/{cluster_id}/consumer-group/{consumer_group_id}/lags`
List of all the lags for a given consumer-group.

{
  "data": [
     {
        "metadata": { link },
        "cluster_id"
        "topic_name"
        "partition_id"
         ...
        "lag"...
     },
  ]
}